compiler/semantic/op.go

```diff
@@ -893,6 +899,21 @@ func (a *analyzer) semConstDecl(c *ast.ConstDecl) (dag.Def, error) {
 	}, nil
 }
 
+func (a *analyzer) semTypeDecl(d *ast.TypeDecl) (dag.Def, error) {
+	typ, err := a.semType(d.Type)
+	if err != nil {
+		return dag.Def{}, err
+	}
+	e := &dag.Literal{
+		Kind:  "Literal",
+		Value: fmt.Sprintf("<%s=%s>", zson.QuotedName(d.Name), typ),
+	}
+	if err := a.scope.DefineConst(a.zctx, d.Name, e); err != nil {
+		return dag.Def{}, err
+	}
+	return dag.Def{Name: d.Name, Expr: e}, nil
```

Comment:
Nit: These are on multiple lines everywhere else in here.
```suggestion
	return dag.Def{
		Name: d.Name,
		Expr: e
	}, nil
```

Reply:
For me I follow my made up rule that if a struct declaration fits below 80 characters, I do it single line.  I think that seems reasonable, no?

Reply:
Totally reasonable. I just prefer to see changes follow the prevailing style. Just a nit though so do as you like!